engine/api/event/event.go

```diff
@@ -0,0 +1,38 @@
+package event
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/docker/docker/pkg/namesgenerator"
+	"github.com/spf13/viper"
+
+	"github.com/ovh/cds/engine/api/cache"
+	"github.com/ovh/cds/engine/log"
+	"github.com/ovh/cds/sdk"
+)
+
+var hostname, cdsname string
+
+// Routine initializes and run event routine dequeue
+func Routine() {
+	var err error
+	hostname, err = os.Hostname()
+	if err != nil {
+		hostname = fmt.Sprintf("Error while getting Hostname: %s", err.Error())
+	}
+	cdsname = namesgenerator.GetRandomName(0)
+
+	withKafka := kafkaRoutine()
```

Comment:
How will we deal with other broker system (rabbitmq, redis, websockets and so on...) ?